plugin/evm/imports_test.go

```diff
@@ -0,0 +1,66 @@
+// (c) 2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package evm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"golang.org/x/tools/go/packages"
+)
+
+// getDependencies takes a fully qualified package name and returns a map of all
+// its recursive package imports (including itself) in the same format.
+func getDependencies(packageName string) (map[string]struct{}, error) {
+	// Configure the load mode to include dependencies
+	cfg := &packages.Config{Mode: packages.NeedDeps | packages.NeedImports | packages.NeedName | packages.NeedModule}
+	pkgs, err := packages.Load(cfg, packageName)
+	if err != nil {
+		return nil, fmt.Errorf("failed to load package %s: %v", packageName, err)
```

Comment:
nit doesn't `package.Load` include the package name in its error already 🤔  (in the best world, it should, maybe not the case haha)

```suggestion
		return nil, fmt.Errorf("failed to load package: %v", err)
```